### DISCUSSION TITLE
community driven screencast

### DISCUSSION THREAD

**Discussion by sukrosono** [chaijs/chai#1399] [General]
As part of the learning software testing, I am aware that good example and brief documentation is key to speed up the learning process. While I am separate testing and documenting a class, What makes great tutorial? comes to mind. That's where the discussion pointed to be!

My axamination is just create good example and give it proper explanation. Add one or two line of comment and give the a TOC.
But is that all, I still missing what part makes tutorial great instead of good?

### Concrete goal
1. Make a standard, more clear recipe
1. Define what's bad explanation or bad example.
1. Define good learning step. there are some concept or prerequisite before start.

Personal expectation (in the future)
- I am able to create screencast that have good standard, that hopefully can be published (listed) on official Chai website.

Intended output
- increase overall Chai popularity.

**Comment by keithamus** [comment]
This is a great idea! I'd be keen to see this happen.

What do you think the next steps should be?

**Reply by sukrosono** [reply]
Currently, I am still not confident enough to do this on my own. What I mean by standard, is our standard. Chai community standard.
Following what's already on Chai website is good enough for now. Creating a screencast based on that maybe is not require less effort but, if there is room for improvement should be addressed. I am also not following up the last 2 years code changes and documentation.

Do you think any changes from v3 till now that have not yet been briefly documented?
To define a good learning step, strongly need input/ review from you guys. The concrete output would be HTML with TOC that is hosted on the official website. My fear most likely comes from key/pre-requisite concepts that might not be related to Chai. I need help on this, also I have an issue with whom and where to reach you guys (dev team) preferably via chat. Maybe on gitter? looks like our slack is dead.

I also need input from my last [screencast](https://www.youtube.com/watch?v=nvN_7iwtYdo). Kinda is I am speaking good enough, etc. Basically just related to my streaming setup.
